sympy/physics/vector/tests/test_frame.py

```diff
@@ -457,6 +457,12 @@ def test_orient_explicit():
     A.orient_explicit(B, eye(3))
     assert A.dcm(B) == Matrix([[1, 0, 0], [0, 1, 0], [0, 0, 1]])
 
+def test_orient_dcm():
+    A = ReferenceFrame('A')
+    B = ReferenceFrame('B')
+    B.orient_dcm(A, eye(3))
+    assert B.dcm(A) == Matrix([[1, 0, 0],[0, 1, 0],[0, 0, 1]])
```

Comment:
This is not a sufficient test because using if you accidently would have used `dcm.T` then the test will still pass. The proposed test in [this PR](https://github.com/sympy/sympy/pull/24768/files) would be better.